src/main/cms/cms_menu_imu.c

```diff
@@ -391,6 +391,73 @@ static CMS_Menu cmsx_menuFilterPerProfile = {
     .entries = cmsx_menuFilterPerProfileEntries,
 };
 
+static uint8_t cmsx_dstPidProfile;
```

Comment:
@mikeller https://github.com/betaflight/betaflight/pull/3601 renames this so something more generic (`DISABLE_EXTENDED_CMS_MENUS`).